modules/mri_violations/php/NDB_Menu_Filter_mri_protocol_check_violations.class.inc

```diff
@@ -47,8 +47,20 @@ class NDB_Menu_Filter_Mri_Protocol_Check_Violations extends NDB_Menu_Filter
      */
     function _setupVariables()
     {
-        $this->query    = " FROM mri_violations_log l LEFT JOIN mri_scan_type m
+        $UID = $_GET["SeriesUID"];
+        //preventing SQL injection -- UID only takes number and dot
+        $pattern = "/^\d[0-9\.]+/";
```

Comment:
This isn't required after adding the $DB->quote call (and if it is, this is only used one place, so you can just use the constant directly in the function call..)